### PR TITLE
darktable - redhat

### PR DESCRIPTION
**Pull request by mccheng7c**

### CODE REVIEW

File: vars/centos-8.yml [hunk 1]
```diff
@@ -14,5 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+_rpm_repository:
```
Comment (hswong3i) [inline vars/centos-8.yml:17]: `_rpm_repository` is used for initializing yum repo with upstream provided `repo.rpm`, not for installing the dependency nor package itself :-(

Comment (hswong3i) [inline vars/centos-8.yml]: `_rpm_repository` is used for initializing yum repo with upstream provided `repo.rpm`, not for installing the dependency nor package itself :-(

Read https://www.darktable.org/install/#rhel and https://www.darktable.org/install/#3rdparty

**Review by hswong3i — CHANGES_REQUESTED** [review]

File: vars/centos-8.yml (hunk 1 above)
Comment (hswong3i) [reply vars/centos-8.yml]: Required pacakge:
- https://centos.pkgs.org/8/centos-powertools-x86_64/lensfun-0.3.2-14.el8.x86_64.rpm.html#howto

Enable PowerTools in our style:
- https://github.com/alvistack/ansible-role-bootstrap/blob/master/vars/centos-8.yml#L42
- https://github.com/alvistack/ansible-role-bootstrap/blob/master/tasks/redhat.yml#L75
- https://github.com/alvistack/ansible-role-kernel/blob/master/vars/centos-8.yml#L36
- https://github.com/alvistack/ansible-role-kernel/blob/master/tasks/redhat.yml#L64

**Review by mccheng7c — COMMENTED** [review]
for 3d6f884, This is not related to the dependency issue, I've made some mistake of git rebase.

**Pull request closed without merging**